[2024-04-13 01:56:51,138][HYDRA] Launching 3 jobs locally
[2024-04-13 01:56:51,138][HYDRA] 	#0 : +exp=exp_6 +site=s1
2024/04/13 01:56:51 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 01:56:51 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 01:56:51,909][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 01:56:51,914][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 01:56:51,915][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 01:56:51,915][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 01:56:51 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
[2024-04-13 01:56:51,926][pytorch_lightning.utilities.rank_zero][INFO] - You are using a CUDA device ('NVIDIA GeForce RTX 4090') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/utilities/model_summary/model_summary.py:454: A layer with UninitializedParameter was found. Thus, the total number of parameters detected may be inaccurate.

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.744
[2024-04-13 01:57:29,399][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.74444 (best 0.74444), saving model to '/tmp/tmp27f19d1c/model_0.ckpt' as top 1
Metric val_f1_score_1 improved by 0.024 >= min_delta = 1e-05. New best score: 0.768
[2024-04-13 01:58:01,568][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.76818 (best 0.76818), saving model to '/tmp/tmp27f19d1c/model_0.ckpt' as top 1
[2024-04-13 01:58:34,505][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' was not in top 1
[2024-04-13 01:59:07,049][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' was not in top 1
[2024-04-13 01:59:40,368][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
[2024-04-13 02:00:13,126][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' was not in top 1
[2024-04-13 02:00:45,898][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 02:01:17,939][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 02:01:50,513][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 02:02:22,957][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 02:02:55,563][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.768. Signaling Trainer to stop.
[2024-04-13 02:03:28,200][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
2024/04/13 02:03:31 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 02:03:31 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 02:03:31 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 02:03:31 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 02:03:31,180][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 02:03:31,180][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 02:03:31,180][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 02:03:31,180][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 02:03:31 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.751
[2024-04-13 02:04:09,449][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.75091 (best 0.75091), saving model to '/tmp/tmp27f19d1c/model_1.ckpt' as top 1
Metric val_f1_score_1 improved by 0.019 >= min_delta = 1e-05. New best score: 0.770
[2024-04-13 02:04:42,423][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.76960 (best 0.76960), saving model to '/tmp/tmp27f19d1c/model_1.ckpt' as top 1
Metric val_f1_score_1 improved by 0.020 >= min_delta = 1e-05. New best score: 0.790
[2024-04-13 02:05:15,185][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' reached 0.78954 (best 0.78954), saving model to '/tmp/tmp27f19d1c/model_1.ckpt' as top 1
Metric val_f1_score_1 improved by 0.006 >= min_delta = 1e-05. New best score: 0.796
[2024-04-13 02:05:48,200][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' reached 0.79559 (best 0.79559), saving model to '/tmp/tmp27f19d1c/model_1.ckpt' as top 1
[2024-04-13 02:06:21,115][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
[2024-04-13 02:06:54,326][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' was not in top 1
[2024-04-13 02:07:27,144][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 02:08:00,511][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 02:08:33,257][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 02:09:06,102][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 02:09:39,224][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
Metric val_f1_score_1 improved by 0.005 >= min_delta = 1e-05. New best score: 0.801
[2024-04-13 02:10:11,816][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' reached 0.80063 (best 0.80063), saving model to '/tmp/tmp27f19d1c/model_1.ckpt' as top 1
Metric val_f1_score_1 improved by 0.004 >= min_delta = 1e-05. New best score: 0.805
[2024-04-13 02:10:45,107][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 1157: 'val_f1_score_1' reached 0.80502 (best 0.80502), saving model to '/tmp/tmp27f19d1c/model_1.ckpt' as top 1
[2024-04-13 02:11:18,112][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 13, global step 1246: 'val_f1_score_1' was not in top 1
[2024-04-13 02:11:51,261][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 14, global step 1335: 'val_f1_score_1' was not in top 1
[2024-04-13 02:12:24,233][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 15, global step 1424: 'val_f1_score_1' was not in top 1
[2024-04-13 02:12:56,528][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 16, global step 1513: 'val_f1_score_1' was not in top 1
[2024-04-13 02:13:29,872][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 17, global step 1602: 'val_f1_score_1' was not in top 1
[2024-04-13 02:14:03,706][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 18, global step 1691: 'val_f1_score_1' was not in top 1
[2024-04-13 02:14:36,123][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 19, global step 1780: 'val_f1_score_1' was not in top 1
[2024-04-13 02:15:09,428][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 20, global step 1869: 'val_f1_score_1' was not in top 1
[2024-04-13 02:15:42,467][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 21, global step 1958: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.805. Signaling Trainer to stop.
[2024-04-13 02:16:14,971][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 22, global step 2047: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 02:16:17 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 02:16:17 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 02:16:17 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 02:16:17 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 02:16:17,645][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 02:16:17,645][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 02:16:17,645][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 02:16:17,645][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 02:16:17 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.676
[2024-04-13 02:16:55,283][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.67637 (best 0.67637), saving model to '/tmp/tmp27f19d1c/model_2.ckpt' as top 1
Metric val_f1_score_1 improved by 0.082 >= min_delta = 1e-05. New best score: 0.758
[2024-04-13 02:17:28,489][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.75844 (best 0.75844), saving model to '/tmp/tmp27f19d1c/model_2.ckpt' as top 1
Metric val_f1_score_1 improved by 0.020 >= min_delta = 1e-05. New best score: 0.779
[2024-04-13 02:18:02,026][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' reached 0.77851 (best 0.77851), saving model to '/tmp/tmp27f19d1c/model_2.ckpt' as top 1
Metric val_f1_score_1 improved by 0.003 >= min_delta = 1e-05. New best score: 0.782
[2024-04-13 02:18:34,633][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' reached 0.78162 (best 0.78162), saving model to '/tmp/tmp27f19d1c/model_2.ckpt' as top 1
Metric val_f1_score_1 improved by 0.015 >= min_delta = 1e-05. New best score: 0.797
[2024-04-13 02:19:07,039][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' reached 0.79701 (best 0.79701), saving model to '/tmp/tmp27f19d1c/model_2.ckpt' as top 1
[2024-04-13 02:19:39,584][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' was not in top 1
[2024-04-13 02:20:12,416][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 02:20:45,313][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 02:21:18,066][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 02:21:51,280][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 02:22:23,794][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
[2024-04-13 02:22:56,807][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
[2024-04-13 02:23:29,393][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 1157: 'val_f1_score_1' was not in top 1
[2024-04-13 02:24:02,019][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 13, global step 1246: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.797. Signaling Trainer to stop.
[2024-04-13 02:24:35,108][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 14, global step 1335: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 02:24:37 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 02:24:37 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 02:24:37 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 02:24:37 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 02:24:37,794][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 02:24:37,794][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 02:24:37,794][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 02:24:37,794][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 02:24:37 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.639
[2024-04-13 02:25:15,944][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.63940 (best 0.63940), saving model to '/tmp/tmp27f19d1c/model_3.ckpt' as top 1
Metric val_f1_score_1 improved by 0.096 >= min_delta = 1e-05. New best score: 0.736
[2024-04-13 02:25:48,417][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.73588 (best 0.73588), saving model to '/tmp/tmp27f19d1c/model_3.ckpt' as top 1
Metric val_f1_score_1 improved by 0.012 >= min_delta = 1e-05. New best score: 0.748
[2024-04-13 02:26:21,382][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' reached 0.74788 (best 0.74788), saving model to '/tmp/tmp27f19d1c/model_3.ckpt' as top 1
Metric val_f1_score_1 improved by 0.020 >= min_delta = 1e-05. New best score: 0.768
[2024-04-13 02:26:54,589][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' reached 0.76832 (best 0.76832), saving model to '/tmp/tmp27f19d1c/model_3.ckpt' as top 1
[2024-04-13 02:27:27,319][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
Metric val_f1_score_1 improved by 0.008 >= min_delta = 1e-05. New best score: 0.776
[2024-04-13 02:28:00,443][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' reached 0.77638 (best 0.77638), saving model to '/tmp/tmp27f19d1c/model_3.ckpt' as top 1
Metric val_f1_score_1 improved by 0.010 >= min_delta = 1e-05. New best score: 0.787
[2024-04-13 02:28:33,463][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' reached 0.78687 (best 0.78687), saving model to '/tmp/tmp27f19d1c/model_3.ckpt' as top 1
[2024-04-13 02:29:06,814][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 02:29:39,606][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 02:30:11,955][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 02:30:44,010][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
[2024-04-13 02:31:16,710][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
[2024-04-13 02:31:49,772][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 1157: 'val_f1_score_1' was not in top 1
[2024-04-13 02:32:22,721][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 13, global step 1246: 'val_f1_score_1' was not in top 1
[2024-04-13 02:32:55,291][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 14, global step 1335: 'val_f1_score_1' was not in top 1
[2024-04-13 02:33:28,472][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 15, global step 1424: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.787. Signaling Trainer to stop.
[2024-04-13 02:34:01,423][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 16, global step 1513: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 02:34:04 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 02:34:04 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 02:34:04 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 02:34:04 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 02:34:04,235][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 02:34:04,235][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 02:34:04,235][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 02:34:04,235][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 02:34:04 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.754
[2024-04-13 02:34:40,983][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.75407 (best 0.75407), saving model to '/tmp/tmp27f19d1c/model_4.ckpt' as top 1
Metric val_f1_score_1 improved by 0.041 >= min_delta = 1e-05. New best score: 0.795
[2024-04-13 02:35:11,841][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.79542 (best 0.79542), saving model to '/tmp/tmp27f19d1c/model_4.ckpt' as top 1
Metric val_f1_score_1 improved by 0.008 >= min_delta = 1e-05. New best score: 0.804
[2024-04-13 02:35:43,000][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' reached 0.80353 (best 0.80353), saving model to '/tmp/tmp27f19d1c/model_4.ckpt' as top 1
[2024-04-13 02:36:14,453][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' was not in top 1
[2024-04-13 02:36:46,586][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
[2024-04-13 02:37:18,322][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' was not in top 1
[2024-04-13 02:37:49,928][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 02:38:22,025][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 02:38:53,545][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 02:39:24,716][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 02:39:55,985][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
[2024-04-13 02:40:27,769][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.804. Signaling Trainer to stop.
[2024-04-13 02:40:59,377][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 1157: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 02:41:02 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 02:41:02 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 02:41:02 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 02:41:02 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 02:41:02,073][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 02:41:02,073][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 02:41:02,073][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 02:41:02,073][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 02:41:02 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.761
[2024-04-13 02:41:38,418][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.76054 (best 0.76054), saving model to '/tmp/tmp27f19d1c/model_5.ckpt' as top 1
Metric val_f1_score_1 improved by 0.018 >= min_delta = 1e-05. New best score: 0.779
[2024-04-13 02:42:10,623][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.77892 (best 0.77892), saving model to '/tmp/tmp27f19d1c/model_5.ckpt' as top 1
Metric val_f1_score_1 improved by 0.010 >= min_delta = 1e-05. New best score: 0.789
[2024-04-13 02:42:42,679][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' reached 0.78880 (best 0.78880), saving model to '/tmp/tmp27f19d1c/model_5.ckpt' as top 1
[2024-04-13 02:43:14,307][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' was not in top 1
[2024-04-13 02:43:46,316][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
[2024-04-13 02:44:18,243][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' was not in top 1
[2024-04-13 02:44:50,211][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 02:45:22,239][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 02:45:54,844][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 02:46:26,318][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 02:46:58,212][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
[2024-04-13 02:47:29,390][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.789. Signaling Trainer to stop.
[2024-04-13 02:48:01,475][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 1157: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 02:48:04 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 02:48:04 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 02:48:04 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 02:48:04 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 02:48:04,164][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 02:48:04,164][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 02:48:04,164][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 02:48:04,164][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 02:48:04 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.733
[2024-04-13 02:48:42,299][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.73348 (best 0.73348), saving model to '/tmp/tmp27f19d1c/model_6.ckpt' as top 1
Metric val_f1_score_1 improved by 0.038 >= min_delta = 1e-05. New best score: 0.772
[2024-04-13 02:49:15,358][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.77175 (best 0.77175), saving model to '/tmp/tmp27f19d1c/model_6.ckpt' as top 1
[2024-04-13 02:49:48,232][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' was not in top 1
[2024-04-13 02:50:20,384][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' was not in top 1
[2024-04-13 02:50:52,638][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
Metric val_f1_score_1 improved by 0.005 >= min_delta = 1e-05. New best score: 0.777
[2024-04-13 02:51:25,103][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' reached 0.77721 (best 0.77721), saving model to '/tmp/tmp27f19d1c/model_6.ckpt' as top 1
Metric val_f1_score_1 improved by 0.007 >= min_delta = 1e-05. New best score: 0.785
[2024-04-13 02:51:57,096][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' reached 0.78470 (best 0.78470), saving model to '/tmp/tmp27f19d1c/model_6.ckpt' as top 1
[2024-04-13 02:52:29,048][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 02:53:01,880][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 02:53:34,317][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 02:54:06,696][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
[2024-04-13 02:54:39,459][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
[2024-04-13 02:55:12,175][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 1157: 'val_f1_score_1' was not in top 1
[2024-04-13 02:55:45,327][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 13, global step 1246: 'val_f1_score_1' was not in top 1
[2024-04-13 02:56:17,485][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 14, global step 1335: 'val_f1_score_1' was not in top 1
[2024-04-13 02:56:50,257][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 15, global step 1424: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.785. Signaling Trainer to stop.
[2024-04-13 02:57:23,324][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 16, global step 1513: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 02:57:25 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 02:57:25 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 02:57:25 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 02:57:25 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 02:57:26,000][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 02:57:26,000][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 02:57:26,000][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 02:57:26,000][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 02:57:26 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.786
[2024-04-13 02:58:03,856][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.78607 (best 0.78607), saving model to '/tmp/tmp27f19d1c/model_7.ckpt' as top 1
[2024-04-13 02:58:37,537][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' was not in top 1
[2024-04-13 02:59:10,720][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' was not in top 1
[2024-04-13 02:59:43,508][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' was not in top 1
[2024-04-13 03:00:15,648][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
[2024-04-13 03:00:48,427][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' was not in top 1
[2024-04-13 03:01:21,670][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 03:01:54,189][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 03:02:27,281][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 03:02:59,729][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.786. Signaling Trainer to stop.
[2024-04-13 03:03:32,744][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 03:03:35 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 03:03:35 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 03:03:35 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 03:03:35 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 03:03:35,420][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 03:03:35,420][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 03:03:35,420][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 03:03:35,420][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 03:03:35 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.682
[2024-04-13 03:04:13,373][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.68163 (best 0.68163), saving model to '/tmp/tmp27f19d1c/model_8.ckpt' as top 1
Metric val_f1_score_1 improved by 0.053 >= min_delta = 1e-05. New best score: 0.735
[2024-04-13 03:04:46,422][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.73484 (best 0.73484), saving model to '/tmp/tmp27f19d1c/model_8.ckpt' as top 1
[2024-04-13 03:05:19,459][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' was not in top 1
[2024-04-13 03:05:51,720][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' was not in top 1
[2024-04-13 03:06:24,463][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' was not in top 1
[2024-04-13 03:06:57,206][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' was not in top 1
[2024-04-13 03:07:30,097][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 03:08:03,721][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 03:08:36,537][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 03:09:09,261][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 03:09:41,710][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.735. Signaling Trainer to stop.
[2024-04-13 03:10:15,194][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 03:10:17 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 03:10:17 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 03:10:17 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 03:10:17 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 03:10:17,877][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 03:10:17,877][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 03:10:17,877][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 03:10:17,877][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 03:10:17 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmp27f19d1c exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.715
[2024-04-13 03:10:54,198][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 89: 'val_f1_score_1' reached 0.71535 (best 0.71535), saving model to '/tmp/tmp27f19d1c/model_9.ckpt' as top 1
Metric val_f1_score_1 improved by 0.066 >= min_delta = 1e-05. New best score: 0.781
[2024-04-13 03:11:25,082][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 178: 'val_f1_score_1' reached 0.78128 (best 0.78128), saving model to '/tmp/tmp27f19d1c/model_9.ckpt' as top 1
Metric val_f1_score_1 improved by 0.010 >= min_delta = 1e-05. New best score: 0.791
[2024-04-13 03:11:55,686][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 267: 'val_f1_score_1' reached 0.79092 (best 0.79092), saving model to '/tmp/tmp27f19d1c/model_9.ckpt' as top 1
Metric val_f1_score_1 improved by 0.017 >= min_delta = 1e-05. New best score: 0.808
[2024-04-13 03:12:26,634][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 356: 'val_f1_score_1' reached 0.80837 (best 0.80837), saving model to '/tmp/tmp27f19d1c/model_9.ckpt' as top 1
Metric val_f1_score_1 improved by 0.003 >= min_delta = 1e-05. New best score: 0.811
[2024-04-13 03:12:57,851][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 445: 'val_f1_score_1' reached 0.81098 (best 0.81098), saving model to '/tmp/tmp27f19d1c/model_9.ckpt' as top 1
Metric val_f1_score_1 improved by 0.004 >= min_delta = 1e-05. New best score: 0.815
[2024-04-13 03:13:28,436][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 534: 'val_f1_score_1' reached 0.81478 (best 0.81478), saving model to '/tmp/tmp27f19d1c/model_9.ckpt' as top 1
[2024-04-13 03:13:59,212][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 623: 'val_f1_score_1' was not in top 1
[2024-04-13 03:14:30,476][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 712: 'val_f1_score_1' was not in top 1
[2024-04-13 03:15:01,732][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 801: 'val_f1_score_1' was not in top 1
[2024-04-13 03:15:32,952][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 890: 'val_f1_score_1' was not in top 1
[2024-04-13 03:16:04,278][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 979: 'val_f1_score_1' was not in top 1
[2024-04-13 03:16:34,974][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 1068: 'val_f1_score_1' was not in top 1
[2024-04-13 03:17:05,785][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 1157: 'val_f1_score_1' was not in top 1
[2024-04-13 03:17:36,696][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 13, global step 1246: 'val_f1_score_1' was not in top 1
[2024-04-13 03:18:07,872][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 14, global step 1335: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.815. Signaling Trainer to stop.
[2024-04-13 03:18:38,894][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 15, global step 1424: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 03:18:41 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 03:18:41 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
[2024-04-13 03:18:41,601][HYDRA] 	#1 : +exp=exp_2 +site=s1
2024/04/13 03:18:41 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 03:18:41 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 03:18:41,691][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 03:18:41,691][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 03:18:41,691][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 03:18:41,691][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 03:18:41 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetSAR        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
2024/04/13 03:18:45 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 03:18:45 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
[2024-04-13 03:18:45,855][HYDRA] 	#2 : +exp=exp_1 +site=s1
2024/04/13 03:18:45 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 03:18:45 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 03:18:45,949][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 03:18:45,949][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 03:18:45,949][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 03:18:45,949][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 03:18:45 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.740
[2024-04-13 03:22:54,880][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.73957 (best 0.73957), saving model to '/tmp/tmpucekida9/model_0.ckpt' as top 1
[2024-04-13 03:26:59,892][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' was not in top 1
[2024-04-13 03:31:02,821][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 03:35:06,066][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 03:39:09,790][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 03:43:14,536][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 03:47:18,716][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 03:51:23,412][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 03:55:27,113][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 03:59:31,406][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.740. Signaling Trainer to stop.
[2024-04-13 04:03:36,253][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 04:03:38 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 04:03:38 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 04:03:38 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 04:03:38 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 04:03:38,962][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 04:03:38,962][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 04:03:38,962][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 04:03:38,962][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 04:03:38 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.822
[2024-04-13 04:07:47,238][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.82173 (best 0.82173), saving model to '/tmp/tmpucekida9/model_1.ckpt' as top 1
[2024-04-13 04:11:50,343][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' was not in top 1
[2024-04-13 04:15:54,556][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 04:19:58,369][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 04:24:02,834][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 04:28:06,536][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 04:32:10,906][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 04:36:15,746][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 04:40:19,076][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 04:44:21,997][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.822. Signaling Trainer to stop.
[2024-04-13 04:48:25,662][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 04:48:28 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 04:48:28 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 04:48:28 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 04:48:28 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 04:48:28,334][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 04:48:28,334][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 04:48:28,334][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 04:48:28,334][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 04:48:28 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.709
[2024-04-13 04:52:38,124][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.70935 (best 0.70935), saving model to '/tmp/tmpucekida9/model_2.ckpt' as top 1
[2024-04-13 04:56:41,777][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' was not in top 1
[2024-04-13 05:00:46,856][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 05:04:50,786][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 05:08:56,276][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 05:13:00,384][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 05:17:05,698][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 05:21:09,876][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 05:25:15,941][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 05:29:20,346][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.709. Signaling Trainer to stop.
[2024-04-13 05:33:25,631][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 05:33:28 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 05:33:28 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 05:33:28 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 05:33:28 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 05:33:28,330][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 05:33:28,330][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 05:33:28,330][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 05:33:28,330][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 05:33:28 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.310
[2024-04-13 05:37:38,567][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.30981 (best 0.30981), saving model to '/tmp/tmpucekida9/model_3.ckpt' as top 1
Metric val_f1_score_1 improved by 0.323 >= min_delta = 1e-05. New best score: 0.633
[2024-04-13 05:41:43,222][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' reached 0.63280 (best 0.63280), saving model to '/tmp/tmpucekida9/model_3.ckpt' as top 1
Metric val_f1_score_1 improved by 0.001 >= min_delta = 1e-05. New best score: 0.634
[2024-04-13 05:45:48,174][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' reached 0.63359 (best 0.63359), saving model to '/tmp/tmpucekida9/model_3.ckpt' as top 1
[2024-04-13 05:49:52,709][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 05:53:56,420][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 05:58:00,583][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 06:02:04,346][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 06:06:08,378][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 06:10:11,948][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 06:14:16,785][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
[2024-04-13 06:18:19,923][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
[2024-04-13 06:22:23,390][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 9540: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.634. Signaling Trainer to stop.
[2024-04-13 06:26:26,742][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 12, global step 10335: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 06:26:29 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 06:26:29 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 06:26:29 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 06:26:29 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 06:26:29,452][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 06:26:29,452][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 06:26:29,452][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 06:26:29,452][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 06:26:29 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.824
[2024-04-13 06:30:38,865][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.82436 (best 0.82436), saving model to '/tmp/tmpucekida9/model_4.ckpt' as top 1
[2024-04-13 06:34:43,593][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' was not in top 1
[2024-04-13 06:38:48,246][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 06:42:54,765][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 06:46:57,914][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 06:51:03,292][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 06:55:08,780][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 06:59:12,422][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 07:03:16,899][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 07:07:22,682][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.824. Signaling Trainer to stop.
[2024-04-13 07:11:26,587][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 07:11:29 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 07:11:29 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 07:11:29 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 07:11:29 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 07:11:29,296][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 07:11:29,296][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 07:11:29,296][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 07:11:29,296][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 07:11:29 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.826
[2024-04-13 07:15:38,996][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.82567 (best 0.82567), saving model to '/tmp/tmpucekida9/model_5.ckpt' as top 1
[2024-04-13 07:19:43,480][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' was not in top 1
[2024-04-13 07:23:48,730][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 07:27:52,303][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 07:31:57,025][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 07:36:01,905][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 07:40:05,262][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 07:44:09,782][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 07:48:14,183][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 07:52:18,247][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.826. Signaling Trainer to stop.
[2024-04-13 07:56:21,566][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 07:56:24 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 07:56:24 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 07:56:24 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 07:56:24 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 07:56:24,299][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 07:56:24,299][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 07:56:24,299][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 07:56:24,299][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 07:56:24 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.509
[2024-04-13 08:00:33,293][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.50857 (best 0.50857), saving model to '/tmp/tmpucekida9/model_6.ckpt' as top 1
Metric val_f1_score_1 improved by 0.195 >= min_delta = 1e-05. New best score: 0.703
[2024-04-13 08:04:38,132][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' reached 0.70344 (best 0.70344), saving model to '/tmp/tmpucekida9/model_6.ckpt' as top 1
[2024-04-13 08:08:43,069][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 08:12:48,542][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 08:16:53,034][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 08:20:57,673][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 08:25:02,077][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 08:29:07,395][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 08:33:13,103][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 08:37:17,498][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
[2024-04-13 08:41:22,146][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.703. Signaling Trainer to stop.
[2024-04-13 08:45:26,948][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 9540: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 08:45:29 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 08:45:29 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 08:45:29 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 08:45:29 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 08:45:29,674][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 08:45:29,674][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 08:45:29,674][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 08:45:29,674][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 08:45:29 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.684
[2024-04-13 08:49:44,725][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.68444 (best 0.68444), saving model to '/tmp/tmpucekida9/model_7.ckpt' as top 1
Metric val_f1_score_1 improved by 0.042 >= min_delta = 1e-05. New best score: 0.727
[2024-04-13 08:53:52,489][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' reached 0.72676 (best 0.72676), saving model to '/tmp/tmpucekida9/model_7.ckpt' as top 1
[2024-04-13 08:58:01,600][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 09:02:09,700][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 09:06:18,440][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 09:10:26,129][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 09:14:35,593][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 09:18:46,348][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 09:22:56,318][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 09:27:05,297][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
[2024-04-13 09:31:15,167][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.727. Signaling Trainer to stop.
[2024-04-13 09:35:24,406][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 9540: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 09:35:27 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 09:35:27 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 09:35:27 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 09:35:27 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 09:35:27,099][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 09:35:27,099][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 09:35:27,099][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 09:35:27,099][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 09:35:27 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.710
[2024-04-13 09:39:41,988][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.70961 (best 0.70961), saving model to '/tmp/tmpucekida9/model_8.ckpt' as top 1
Metric val_f1_score_1 improved by 0.068 >= min_delta = 1e-05. New best score: 0.777
[2024-04-13 09:43:51,615][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' reached 0.77726 (best 0.77726), saving model to '/tmp/tmpucekida9/model_8.ckpt' as top 1
[2024-04-13 09:48:03,027][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 09:52:12,137][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 09:56:21,725][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 10:00:30,690][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 10:04:40,153][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 10:08:48,484][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 10:12:58,208][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 10:17:07,374][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
[2024-04-13 10:21:16,288][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.777. Signaling Trainer to stop.
[2024-04-13 10:25:25,168][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 11, global step 9540: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 10:25:27 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 10:25:27 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
2024/04/13 10:25:27 INFO mlflow.system_metrics.system_metrics_monitor: Started monitoring system metrics.
2024/04/13 10:25:27 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of pytorch. If you encounter errors during autologging, try upgrading / downgrading pytorch to a supported version, or try upgrading MLflow.
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
[2024-04-13 10:25:27,884][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-04-13 10:25:27,884][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-04-13 10:25:27,884][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-04-13 10:25:27,884][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
2024/04/13 10:25:27 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py:463: UserWarning: Autologging is known to be compatible with pytorch-lightning versions between 1.4.9 and 2.2.0.post0 and may not succeed with packages outside this range."
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:653: Checkpoint directory /tmp/tmpucekida9 exists and is not empty.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name         | Type              | Params
---------------------------------------------------
0 | model        | ResUnetOpt        | 0     
1 | criterion    | FocalLoss         | 0     
2 | pred_softmax | Softmax           | 0     
3 | train_metric | MulticlassF1Score | 0     
4 | val_metric   | MulticlassF1Score | 0     
---------------------------------------------------
0         Trainable params
0         Non-trainable params
0         Total params
0.000     Total estimated model params size (MB)
Metric val_f1_score_1 improved. New best score: 0.794
[2024-04-13 10:29:41,628][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 0, global step 795: 'val_f1_score_1' reached 0.79376 (best 0.79376), saving model to '/tmp/tmpucekida9/model_9.ckpt' as top 1
[2024-04-13 10:33:48,376][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 1, global step 1590: 'val_f1_score_1' was not in top 1
[2024-04-13 10:37:55,603][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 2, global step 2385: 'val_f1_score_1' was not in top 1
[2024-04-13 10:42:02,268][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 3, global step 3180: 'val_f1_score_1' was not in top 1
[2024-04-13 10:46:09,727][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 4, global step 3975: 'val_f1_score_1' was not in top 1
[2024-04-13 10:50:15,648][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 5, global step 4770: 'val_f1_score_1' was not in top 1
[2024-04-13 10:54:24,288][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 6, global step 5565: 'val_f1_score_1' was not in top 1
[2024-04-13 10:58:31,191][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 7, global step 6360: 'val_f1_score_1' was not in top 1
[2024-04-13 11:02:38,470][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 8, global step 7155: 'val_f1_score_1' was not in top 1
[2024-04-13 11:06:45,727][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 9, global step 7950: 'val_f1_score_1' was not in top 1
Monitored metric val_f1_score_1 did not improve in the last 10 records. Best score: 0.794. Signaling Trainer to stop.
[2024-04-13 11:10:53,879][pytorch_lightning.utilities.rank_zero][INFO] - Epoch 10, global step 8745: 'val_f1_score_1' was not in top 1
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/lazy.py:181: UserWarning: Lazy modules are a new feature under heavy development so changes to the API or functionality can happen at any moment.
  warnings.warn('Lazy modules are a new feature under heavy development '
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:11: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/_distutils_hack/__init__.py:26: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
2024/04/13 11:10:56 INFO mlflow.system_metrics.system_metrics_monitor: Stopping system metrics monitoring...
2024/04/13 11:10:56 INFO mlflow.system_metrics.system_metrics_monitor: Successfully terminated system metrics monitoring!
Error executing job with overrides: ['+exp=exp_2', '+site=s1']
Traceback (most recent call last):
  File "/home/felferrari/projects/thesis2/train.py", line 78, in train
    trainer.fit(
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/utils/autologging_utils/safety.py", line 578, in safe_patch_function
    patch_function(call_original, *args, **kwargs)
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/utils/autologging_utils/safety.py", line 251, in patch_with_managed_run
    result = patch_function(original, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/pytorch/_lightning_autolog.py", line 537, in patched_fit
    result = original(self, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/utils/autologging_utils/safety.py", line 559, in call_original
    return call_original_fn_with_event_logging(_original_fn, og_args, og_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/utils/autologging_utils/safety.py", line 494, in call_original_fn_with_event_logging
    original_fn_result = original_fn(*og_args, **og_kwargs)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/mlflow/utils/autologging_utils/safety.py", line 556, in _original_fn
    original_result = original(*_og_args, **_og_kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/trainer/trainer.py", line 544, in fit
    call._call_and_handle_interrupt(
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/trainer/call.py", line 44, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/trainer/trainer.py", line 580, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/trainer/trainer.py", line 987, in _run
    results = self._run_stage()
              ^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/trainer/trainer.py", line 1031, in _run_stage
    self._run_sanity_check()
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/trainer/trainer.py", line 1060, in _run_sanity_check
    val_loop.run()
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/loops/utilities.py", line 182, in _decorator
    return loop_run(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/loops/evaluation_loop.py", line 135, in run
    self._evaluation_step(batch, batch_idx, dataloader_idx, dataloader_iter)
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/loops/evaluation_loop.py", line 396, in _evaluation_step
    output = call._call_strategy_hook(trainer, hook_name, *step_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/trainer/call.py", line 309, in _call_strategy_hook
    output = fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/lightning/pytorch/strategies/strategy.py", line 412, in validation_step
    return self.lightning_module.validation_step(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/projects/thesis2/src/models/model_module.py", line 104, in validation_step
    x = self.model.prepare(x)
        ^^^^^^^^^^^^^^^^^^
  File "/home/felferrari/anaconda3/envs/thesis2/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1688, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'ResUnetSAR' object has no attribute 'prepare'

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
